### PR TITLE
Including Python layer file during importing models

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
Download python layer file if specified in info.json
Pretrained model copies python layer file
Automatically fill server-side python layer file name when users click 'Customize'

### CODE REVIEW

File: digits/pretrained_model/job.py [hunk 1]
```diff
@@ -47,6 +47,16 @@ def get_weights_path(self):
     def get_model_def_path(self):
         return self.tasks[0].get_model_def_path()
 
+    def get_python_layer_path(self):
+        tmp_dir = os.path.dirname(self.tasks[0].get_model_def_path())
+        python_layer_file_name = 'digits_python_layers.py'
```
Comment (lukeyeager) [inline digits/pretrained_model/job.py]: Don't we have a constant for this filename somewhere?

Comment (lukeyeager) [inline digits/pretrained_model/job.py]: Ok. We can fix it later I guess.

Comment (IsaacYangSLA) [inline digits/pretrained_model/job.py]: Yes.  It can be done when we spend sometime refactoring my messy codes.

**Comment by IsaacYangSLA** [comment]
I will merge it after Model Store UI improvement is merged.

**Comment by IsaacYangSLA** [comment]
Merged manually with commandline.

**Pull request closed without merging**